src/views/IncidentDetails.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <center-wrapper>
+    <btn-back title="Back to the list" />
+    <incident-details :incident="incident" />
```

Comment:
Nit: do we benefit at all be having 2 `IncidentDetail` files? One is a view and the other is component. But the component is only used in one place.

Reply:
Agree. I have renamed the views/incidentDetails => views/incident. This might still be confusing, but the component/incidentDetails is now a dumb component. 
(in the other branch though).